appveyor.yml

```diff
@@ -1,37 +1,23 @@
 environment:
 
-  CONDA_INSTALL_LOCN: "C:\\conda"
-
   # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
   # /E:ON and /V:ON options are not enabled in the batch script interpreter
   # See: http://stackoverflow.com/a/13751649/163740
   CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
 
-  # Workaround for https://github.com/conda/conda-build/issues/636
-  PYTHONIOENCODING: "UTF-8"
-  PYTHONUNBUFFERED: "1"
-
-  # We set a default Python version for the miniconda that is to be installed.
-  # This can be overridden in the matrix definition where appropriate.
-  CONDA_PY: "27"
-
   matrix:
-    - TARGET_ARCH: x86
-      CONDA_NPY: 111
-      CONDA_PY: 27
```

Comment:
I updated the configuration and remove the `x86` tests and Python 3.5. I believe that `x64` and both Python 2.7 and 3.6 is enough but let me know if you disagree and I can add them back.